packages/manager/apps/pci-gateway/src/components/size/SizeLabel.component.tsx

```diff
@@ -0,0 +1,77 @@
+import { useTranslation } from 'react-i18next';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import {
+  useCatalogPrice,
+  convertHourlyPriceToMonthly,
+} from '@ovh-ux/manager-react-components';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import { TProductAddonDetail } from '@/types/addon.type';
+
+export default function SizeLabel({
+  size,
+  bandwidth,
+  price,
+}: Readonly<TProductAddonDetail>) {
+  const { t } = useTranslation('catalog-selector');
+
+  const {
+    getFormattedHourlyCatalogPrice,
+    getFormattedMonthlyCatalogPrice,
+  } = useCatalogPrice(4);
+
+  return (
+    <div className="grid grid-cols-1 gap-2 text-left text w-full">
+      <div className="">
+        <OsdsText
+          level={ODS_THEME_TYPOGRAPHY_LEVEL.heading}
+          size={ODS_THEME_TYPOGRAPHY_SIZE._200}
+          color={ODS_THEME_COLOR_INTENT.text}
+        >
+          {t('pci_projects_project_gateways_model_selector_size')}{' '}
+          {size.toUpperCase()}
```

Comment:
Why not use translation with parameter ? 
```suggestion
          {t('pci_projects_project_gateways_model_selector_size', {size: size.toUpperCase()} )}
```